torch/lib/c10d/ProcessGroupGloo.hpp

```diff
@@ -89,8 +93,13 @@ class ProcessGroupGloo : public ProcessGroup {
 
     virtual void run() = 0;
 
+    std::vector<at::Tensor> result() override;
+
    protected:
     friend class ProcessGroupGloo;
+
+   private:
+    std::vector<std::vector<at::Tensor>> outputTensors_;
```

Comment:
You use `const std::vector<std::vector<at::Tensor>>`, to make sure the length this 2D vector won't change. This will still allow the update of the underlying tensor data.

Reply:
done.